packages/e2e-tests/specs/block-deletion.test.js

```diff
@@ -34,8 +34,8 @@ describe( 'block deletion -', () => {
 			// The blocks can't be empty to trigger the toolbar
 			await page.keyboard.type( 'Paragraph to remove' );
 
-			// Move the mouse to show the block toolbar
-			await page.mouse.move( 200, 300, { steps: 10 } );
```

Comment:
Last time I tried this change, the mouse moving was still more reliable but I don't remember the details, maybe it's better this time :)

Reply:
> Last time I tried this change, the mouse moving was still more reliable but I don't remember the details, maybe it's better this time :)

I can't imagine why that would be 🤷‍♂️ The tests are passing here at least. Let me know if you can find a reference to the issues you'd seen previously.

Reply:
I don't remember precisely and I agree it's better using the keyboard shortcuts. The intent is clearer.